### PR TITLE
Added Kabyle, see issue on CrowdIn

### PR DESCRIPTION
**Pull request by ricardobalk**
Got a message on CrowdIn﻿ that the language Kabyle ("Taqbaylit" in local name) was not available to choose in the language list within the ProtonMail Android app.

The original message is as follows:

> Hello,
> I just installed the latest version of the Android application, to have the interface of ProtonMail in Kabyle we have to adjust the  language of the phone on Kabyle. The Kabyle is absent in the language selection menu. In examining the source code, I 
> identified the problem, there are no entries for the Kabyle language in the array file.xml that is on the folder: res/values. I would like to see that fixed, thank you.

Although I don't know if this fixes it, I thought: let's help by submitting a PR directly to the repository.

Kind regards,

Ricardo

### CODE REVIEW

**Comment by Azwaw1** [comment]
Hello, the native name of kabyle is "Taqbaylit" not Izwawen.

File: app/src/main/res/values/cnfg.xml [hunk 1]
```diff
@@ -50,6 +50,7 @@ along with ProtonMail. If not, see https://www.gnu.org/licenses/.
         <item>íslenska</item>
         <item>Italiano</item>
         <item>日本語</item>
+        <item>Izwawen</item>
```
Comment (Aksel103) [inline app/src/main/res/values/cnfg.xml]: Hello, The native name of kabyle is "Taqbaylit" note kabyle, please, would you fix that, than you.

**Review by Azwaw1 — APPROVED** [review]

File: app/src/main/res/values/cnfg.xml (hunk 1 above)
Comment (ricardobalk) [reply app/src/main/res/values/cnfg.xml]: My apologies, I was not aware of that. I fixed it with an additional commit. Wish you a nice day! :)

**Comment by ricardobalk** [comment]
Just tested it in Android Studio as well, it looks like this now:

image

image

Also my apologies to PM for not creating a separate branch on my fork, I was not aware of that at the time I made my Git commits. There's also another PR for changing "Dutch" to "Nederlands", which I created in a separate branch (aka how it needs to be done).

**Comment by Azwaw1** [comment]
In the latest version of ProtonMail Android, Kabyle (Taqbaylit) is not yet available in the list of languages in the settings.

**Comment by Azwaw1** [comment]
please any one to merge???

**Comment by Azwaw1** [comment]
I just downloaded the latest Android beta, still no Kabyle language in the language selection in the settings, please someone could merge this PR ????

**Comment by Azwaw1** [comment]
@ricardobalk ???

**Comment by ricardobalk** [comment]
@SelyanKab I'm not hired as an active employee at @ProtonMail, so there's not much I can do.

**Comment by kylekatarnls** [comment]
Hello, I'll try to see the status for this. Maybe we have some blocker.

**Comment by Azwaw1** [comment]
@ricardobalk , @kylekatarnls Thank you.

**Comment by dhoko** [comment]
Hi 👋, 

A permanent fix is going to be available monday for this issue. The l10n team was not aware of this file that's why the file is not up-to-date + contains wrong labels.
Bonus it's also missing :greece: locale.

We're going to link it to our jobs providing translations, which is going to allow us to auto provide the correct labels for locales as you did here. 

Sorry for the delay. :/

**Comment by ricardobalk** [comment]
Thanks, @kylekatarnls and @dhoko 👍🏼 

Does that also apply to #90? There, the label `Dutch` should be `Nederlands`, when using its localized name, like the others.

Anyway, thanks for getting back to this. Have a great weekend!

**Comment by ZiriSut** [comment]
Hello,
I don't understand why Kabyle is still not in the language selection in the settings. I just downloaded the latest version of Android, but still no Kabyle language.
Regards.

**Comment by Azwaw1** [comment]
Thank you @kylekatarnls and @dhoko

**Comment by MohammedBelkacem** [comment]
Finally merged? @SelyanKab

**Comment by kylekatarnls** [comment]
> A permanent fix is going to be available monday

Monday :)

**Comment by Azwaw1** [comment]
@mozillakab Monday

**Comment by dhoko** [comment]
@ricardobalk Yes, now this file is linked to our bot. It will automatically bind the correct names for all the locales inside the app.

@ZiriSut idk when the new version is going to be out (we need to QA etc.)

**Comment by ZoricaStojchevska** [comment]
Hi, this has been addressed by our l10n team as @dhoko said in the comments above. You can now find the language under `Taqbaylit`. So we are closing this PR. Thank you for your contribution.

**Pull request closed without merging**